src/app/user/user-form.component.html

```diff
@@ -38,6 +38,12 @@
         maxlength="10">
     </md-input-container>
   </div>
+
+  <div class="flex row">
+    <md-checkbox [(ngModel)]="model.subscribeTestimonials" name="subscribeTestimonials">
```

Comment:
It's inside a container that only shows for admins https://github.com/Sirimangalo/meditation-plus-angular/blob/044d03124f0e49a84e8f5a6dfcc55cd50c2ae0e5/src/app/user/user-form.component.html#L15